__tests__/app.test.js

```diff
@@ -23,4 +23,40 @@ describe('/api/topics', () => {
                 
             })
     })
-})
\ No newline at end of file
+})
+
+describe('/api/articles/:article_id', () => {
+    test('200: responds with correct article', () => {
+        return request(app).get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+                expect(response.body).toEqual({
+                article_id:1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: '2020-07-09T20:11:00.000Z',
+                votes: 100,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              })
+        })
+    })
```

Comment:
Would be good to have a flexible matches here for all properties apart from the article_id - this will keep your test flexible so that it would pass if the data were to be slightly changed